### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by Magisus**
I found these while running `server` tests on the FIPS platform. It turns out they're not exactly FIPS related, but are still good cleanup.

Once these are cleaned up, all `server` tests passed on the FIPS platform, with the caveat that we are not yet using the FIPS version of BouncyCastle. However, with these fixes (notably the tag-related one), we should be able to enable testing on redhatfips in the server pipeline.

### CODE REVIEW

File: acceptance/tests/ssl/certificate_extensions.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@
 test_name "certificate extensions available as trusted data" do
   confine :except, :platform => /^cisco_/ # See PUP-5827
 
+  skip_test "Test requires at least one non-master agent" if hosts.length == 1
+
   tag 'audit:high',        # ca/cert core functionality
       'audit:integration',
       'server'             # Ruby implimentation is deprecated
```
Comment (kris-bosland) [inline acceptance/tests/ssl/certificate_extensions.rb]: How do you feel about a drive-by typo fix here?  `implimentation` -> `implementation`

Comment (Magisus) [reply acceptance/tests/ssl/certificate_extensions.rb]: I'll fix it
